### PR TITLE
Add support for Select all for regions in country modal desktop

### LINKED ISSUE #1517: User Edit: Country Role select/unselect all by region
Add check select all / unselect all next to each region name that allows selecting or unselecting all countries in that region

OJ: When assigning countries to a user (reviewer, collaborator) ad possibility to select all countries for a region or select all regions for the world

### PR DESCRIPTION
**Pull request by sorja**
Resolves #1517 
- [x] Desktop
- [x] Mobile

Note:
Feature hides Select All/Unselect All when all countries of region are selected under another role

image

image

### CODE REVIEW

**Review by rfontanarosa — APPROVED** [review]

**Review by minotogna — CHANGES_REQUESTED** [review]
Select / unselect all should be visible always and select / deselect enabled countries .

Plus , can the button be located on the right of the region label ?

**Review by minotogna — CHANGES_REQUESTED** [review]
move  select/all unselect all next to the label with 4/8 px distance

**Comment by sorja** [comment]
> move select/all unselect all next to the label with 4/8 px distance

image

File: src/client/styles/checkbox.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'src/client/styles/config';
 
 .fra-checkbox {
-  width: 14px;
+  min-width: 14px;
```
Comment (minotogna) [inline src/client/styles/checkbox.scss]: why should be this bigger than 14px?

Comment (sorja) [reply src/client/styles/checkbox.scss]: There was weird issue where the checkbox would clip to 12px with `width` but would work with `min-width`; seems ot be fixed now so reverted back

**Review by minotogna — APPROVED** [review]

**Pull request merged**